apps/prairielearn/elements/pl-matching/pl-matching.css

```diff
@@ -51,3 +51,8 @@
 .pl-matching-container-bottom {
   display: block;
 }
+
+.noedit {
+  opacity: 0.6;
+  pointer-events: none;
+}
```

Comment:
I think this is overkill. Just setting the `select` to `disabled` is enough to change the background color (which achieves enough visual effect), and gets rid of pointer events. Also, none of the other elements do any additional formatting in this case.

Reply:
Oh okay, I will get rid of this.